.vitepress/constants/arabica_versions.js

```diff
@@ -3,7 +3,7 @@ const arabicaVersions = Object.freeze({
   "app-latest-sha": "48173df3dc78f9348eedb3796f29ef9e9e5dc584",
   "core-latest-tag": "v1.40.1-tm-v0.34.29-rc0",
   "core-latest-sha": "0d2b63836d0f4587e162bfded58f53fba238e69c",
-  "node-latest-tag": "v0.16.0-rc0",
-  "node-latest-sha": "603288597fbd82c44aaa0a0ac8d187cac1860431",
+  "node-latest-tag": "v0.16.0",
+  "node-latest-sha": "6744f648649ebb5fee1b27faf7aca96ecf4519b2",
```

Comment:
Arabica is still on v0.16.0-rc0

![Screenshot 2024-08-30 at 10 16 40 AM](https://github.com/user-attachments/assets/32ef9740-ed17-464c-9902-465aaf90b466)

but celestia-node asked DevOps to upgrade so we can hold off on merging this until DevOps upgrades.